pages/landing/hipaa/template/pricing/index.html

```diff
@@ -18,31 +18,31 @@
     description:
       Early access subscribers will receive a CIS staging environment when invited<br>
       and a fully HIPAA-compliant production infrastructure when it’s ready.
-    price: 3060
-    strike_price: 10000
-    flat_fee: 7425
-    strike_flat_fee: 9900
+    price: "3,060"
```

Comment:
That seems rather cheap. A CIS sub goes for $2,040/mo, and customers who need it buy it without too much argument. HIPAA seems like a much bigger, more important, more public facing, audited standard, so I was suggesting ~5x the cost for it, or $10K/mo. Obviously, this should be discounted from that, but should we really discount it by ~70%? That seems to devalue the original sub and the data we'll get from this experiment.

Perhaps we do something like:

- Original price: $9,950/year + $10K up-front
- Discount price: $5,950/year + $0 up-front

Reply:
Updated in 83985751.